Update landing pages (#3368)
canonical/ubuntu-pro-client: docs/references.rst

***************************

Our reference section contains technical information for the Ubuntu Pro Client.

System and other requirements
=============================

> The Ubuntu Pro Client is lightweight and has very few requirements. 

Reviewer: If this is true, does this need to be a standalone sub-section?

Author: Probably not, but I wasn't sure where else to put it :upside_down_face: in the absence of a better place I'll leave it in situ for now

* :ref:`Network requirements <network_requirements>`

Interacting with the Client
===========================

There are two main ways to interact with the Pro Client: on the command line
interface (CLI) in your terminal, or automatically via the API.

* :ref:`CLI reference guide <commands>`
* :ref:`API reference guide <api>`

> You may, in a production environment, also wish to test upcoming changes. For
> this, you can use the available Personal Package Archives (PPA) for the Ubuntu
> Pro Client.

Reviewer 2: I see what you're saying, but technically you would want to test _not_ in a production environment. 
```suggestion
You may, in a staging or canary environment, also wish to test upcoming changes. For
this, you can use the available Personal Package Archives (PPA) for the Ubuntu
Pro Client.
```

Author: Yes, you're quite right - that was imprecise of me

* :ref:`Personal Package Archives (PPAs) <PPAs>`

> You may occasionally encounter an error message when interacting with the
> Pro Client. If you do, this page will help you to resolve the error.

Reviewer 2: This page only has a couple errors that are still possible to encounter, but anyone who has upgraded in the past year or so should never see them. We may need to consider removing this page.
```suggestion
You may occasionally encounter an error message when interacting with the
Pro Client. If you do, check if this page has your specific error.
```
